libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationViewModel.java

```diff
@@ -265,15 +259,13 @@ void sendEventFailedReroute(String errorMessage) {
     return destination;
   }
 
-  private void initializeConnectivityManager(Application application) {
-    connectivityManager = (ConnectivityManager) application.getSystemService(Context.CONNECTIVITY_SERVICE);
+  private void initializeRouter() {
+    RouteFetcher onlineRouter = new RouteFetcher(getApplication(), accessToken);
```

Comment:
should this be `RouteFetcher routeFetcher`?

Reply:
I named this `onlineRouter` to coincide with the `offlineRouter`, just thought it was more clear `onlineRouter` vs. `offlineRouter` than `routeFetcher` vs. `offlineRouter`.